virtcontainers/qemu.go

```diff
@@ -921,6 +923,9 @@ func (q *qemu) togglePauseSandbox(pause bool) error {
 }
 
 func (q *qemu) qmpSetup() error {
+	q.qmpMonitorCh.Lock()
+	defer q.qmpMonitorCh.Unlock()
+
```

Comment:
If we get parallel qmp setup, is it possible to get parallel qmp shutdown as well?

Reply:
ping. The commit message says "Solve possible race condition in qmpSetup() and qmpShutdown()" but the shutdown part seems missing?

Reply:
odd... I have already pushed that commit. Anyway, I do it again.  Thanks :)